tensorflow/python/training/saver.py

```diff
@@ -1390,20 +1390,20 @@ def import_meta_graph(meta_graph_or_file,
   NOTE: Restarting training from saved `meta_graph` only works if the
   device assignments have not changed.
 
-  Example 2:
+  Example:
   Variables, placeholders, and independent operations can also be stored, as
   shown in the following example.
 
   ```Python
   # Saving contents and operations.
   v1 = tf.compat.v1.placeholder(tf.float32, name="v1")
   v2 = tf.compat.v1.placeholder(tf.float32, name="v2")
-  v3 = tf.mul(v1, v2)
+  v3 = tf.multiply(v1, v2)
   vx = tf.Variable(10.0, name="vx")
   v4 = tf.add(v3, vx, name="v4")
   saver = tf.compat.v1.train.Saver([vx])
   sess = tf.compat.v1.Session()
-  sess.run(tf.compat.v1.initialize_all_variables())
+  sess.run(tf.compat.v1.global_variables_initializer())
```

Comment:
Would you be interested in upgrading this example to TF 2.0 style (removing all instances of `tf.compat.v1`)?

Reply:
Yes. I am interested in upgrading this to TF2.0. Please guide me to the location. Currently I am searching but if you see this message, Please let me know the location where to update this example. Thanks!

Reply:
Hi Paige
Is this the page that needs to be updated for 2.0?
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/train/Checkpoint

Thanks

Reply:
Any update on this? Ideally we would like the example in code to not contain `tf.compat.v1`

Reply:
Updated for TF1.13.1 ([gist is here](https://colab.sandbox.google.com/gist/jvishnuvardhan/5fd4552ef7beadd139907d9c1b15296a/saver.ipynb)). Please let me know if there is anything need to be updated.